backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepositoryCache.java

```diff
@@ -18,6 +18,7 @@
 import com.virtuslab.gitmachete.backend.api.IGitMacheteRepositoryCache;
 import com.virtuslab.gitmachete.backend.impl.hooks.PreRebaseHookExecutor;
 import com.virtuslab.gitmachete.backend.impl.hooks.StatusBranchHookExecutor;
+import com.virtuslab.gitmachete.backend.impl.repository.GitMacheteRepository;
 
 public class GitMacheteRepositoryCache implements IGitMacheteRepositoryCache {
```

Comment:
Should this class go to `repository` subpackage? Or maybe the `repository` subpackage isn't really needed?